data/human/news.txt

```diff
@@ -184,6 +185,7 @@ news "human mining"
 			"I buried my dad last week. He was a miner, too. The company didn't give me any time off so I had to pay to attend my own father's funeral. Can you believe that?"
 			"A few years of this and I'll be able to afford my own starship. I pulled in more than 120,000 credits last year! That's more than my brother makes on Hermes, and he's a scientist."
 			"My team is sinking in the rankings again. I should have known better than to put 500 credits on them when they had such a bad season opener."
+			"I've heard that miners who work on Clink are thirteen orders of magnitude more likely to develop cancer. I must admit that the wages are mighty tempting, though..."
```

Comment:
These are miners not politicians or scientists, perhaps `...are thirteen times more likely...` would be a better fit?

Reply:
Yeah. Also thirteen orders of magnitude is like a quadrillion or something, and being that much more likely to happen seems unlikely. :) If it really was that much more likely, the average person would explain it as "all miners who work on clink experience it".

Reply:
Yeah, for example, modern mining operations still have a 0.5-1% incidence of cancer.
2 orders of magnitude would make that 50-100%, 3 would guarantee everybody.